### ISSUE TITLE
Adding Name via JavaScript API Wraps Formula in Double Quotes

### ISSUE THREAD

**Issue by GregReddick** [OfficeDev/office-js#2283]
Adding a name via the JavaScript API forms the address part of the name wrong, surrounding the address with double quote marks.

## Your Environment
Windows 11, Microsoft® Excel® for Microsoft 365 MSO (Version 2110 Build 16.0.14527.20270) 64-bit 
 
## Expected behavior
Address should be without quotes.

## Steps to reproduce
1.  Yo Office, Task Pane Project, TypeScript, Excel
2. Replace the run code in /src/taskpane.ts with this code:
```
export async function run() {
  try {
    await Excel.run(async (context) => {
      let wks: Excel.Worksheet = context.workbook.worksheets.add("Excel Name Bug");
      wks.activate();
      wks.names.add("Foo", "'Excel Name Bug'!$A$1");
      await context.sync();
    });
  } catch (error) {
    console.error(error);
  }
}
```
3. Start the code, Open the Task Pane, Click Run.
4. Click Formulas > Name Manager on the ribbon, examine the "Refers To" field for Foo

Result: the value in the name manager is
```
="'Excel Name Bug'!$A$1"
```
when it should be
```
='Excel Name Bug'!$A$1
```
In other words, there are double quotes surrounding the formula, which means that Foo returns the literal string
```
="'Excel Name Bug'!$A$1"
```
instead of the value in cell A1.

5. Place a value in cell A1.
6. Go to any other cell and enter the formula
```
=Foo
```
to see how this is wrong.

**Comment by GregReddick** [comment]
Never mind...my bug. If I put an equals sign in front of the formula, it does the right thing. Sorry.
